docs/sources/api/_index.md

```diff
@@ -609,6 +610,21 @@ running Loki on Kubernetes, `/ready` can be used as a readiness probe.
 
 In microservices mode, the `/ready` endpoint is exposed by all components.
 
+## Change log level at runtime
+
+```
+GET /log_level
+POST /log_level
+```
+
+`/log_level` a `GET` returns the current log level and a `POST` lets you change the log level of a Loki process at runtime. This can be useful for accessing debugging information during an incident. Caution should be used when running  at the `debug` log level, as this produces a large volume of data.
```

Comment:
```suggestion
A `GET` returns the current log level and a `POST` lets you change the log level of a Loki process at runtime. This can be useful for accessing debugging information during an incident. Caution should be used when running at the `debug` log level, as this produces a large volume of data.
```

@JStickler I accepted your original suggestion. Would it be better to remove the endpoint from the beginning? Or use a hyphen or colon to separate the endpoint from the description. What would you prefer?